x-pack/plugins/security_solution/public/management/components/endpoint_responder/endpoint_response_actions_console_commands.ts

```diff
@@ -23,7 +23,7 @@ import type {
 import { getCommandAboutInfo } from './get_command_about_info';
 
 const emptyArgumentValidator = (argData: ParsedArgData): true | string => {
-  if (argData?.length > 0 && argData[0]?.trim().length > 0) {
+  if (argData?.length > 0 && 'string' === typeof argData[0] && argData[0]?.trim().length > 0) {
```

Comment:
nit: I feel like it's more standard to reverse this `typeof argData[0] === 'string'`.